modules/gitops-configure-rollout-route-traffic-using-openshift-routes.adoc

```diff
@@ -0,0 +1,253 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-configure-rollout-route-traffic-using-openshift-routes_{context}"]
+= Configuring Argo Rollouts to route traffic by using OpenShift Routes
+
+You can configure Argo Rollouts to route traffic by using OpenShift Routes. In this example, create a Route, a Rollout, and two Services, to incrementally route an increasing percentage of traffic, for example from 30% to 60% and then 100%, to a canary version of the application before that canary state is marked as successful and becomes the new stable version.
+
+.Prerequisites
+
+* You have logged in to the {OCP} cluster as an administrator.
+* You have installed the {gitops-title} on your {OCP} cluster.
+* You have installed Argo Rollouts on your {OCP} cluster. For more information, see Creating a RolloutManager custom resource.
+* You have installed the {gitops-title} CLI on your system. For more information, see Installing the GitOps CLI. 
+* You have installed the Argo Rollouts CLI on your system. For more information, see Configuring the GitOps CLI.
+
+.Procedure 
+
+. Create a Route. 
+.. In the Administrator perspective of the web console, click *Networking* -> *Routes*.
+.. Click *Create Route*.
+.. On the *Create Route* page, click *YAML view* and add the following snippet:
++
+.Example: Creating a `Route`
+[source,yaml]
+----
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: rollouts-demo-route #<1>
+spec:
+  port:
+    targetPort: http
+  tls:
+    insecureEdgeTerminationPolicy: Redirect
+    termination: edge
+  to:
+    kind: Service
+    name: argo-rollouts-stable-service #<2>
+    weight: 100 # this field will be automatically modified by Route Rollout plugin, with stable weight
+
+  alternateBackends:
+    - kind: Service
+      name: argo-rollouts-canary-service #<3>
+      weight: 0 # this field will be automatically modified by Route Rollout plugin, with canary weight
+----
+<1> Name of the route - rollouts-demo-route
+<2> Name of the stable service - argo-rollouts-stable-service
+<3> Name of the canary service - argo-rollouts-canary-service
++
+.. Click Create to create the Route. A route, *rollouts-demo-route*, is displayed.
+. Create the services, canary and stable, to be referenced in the Route.
+.. In the Administrator perspective of the web console, click *Networking* -> *Services*.
+.. Click *Create Service*.
+.. On the *Create Service* page, click *YAML view* and add the following snippet:
++
+.Example: Creating a `Canary service`
+[source,yaml]
+----
+# Canary traffic is directed to this Service, for example, it might initially be 30% of traffic
```

Comment:
Again, are these comments a part of the YAML or are they provided by the SME?
If they are a part of the YAMl then it can stay like this. But if these are comments added by SME, we should try to convey the information in a different way (e.g. callouts, example explanation, ...)

Reply:
I am keeping this comment as is since no further changes are required.